runtime/interpreter/errors.go

```diff
@@ -395,21 +395,6 @@ func (e TypeLoadingError) Error() string {
 	return fmt.Sprintf("failed to load type: %s", e.TypeID)
 }
 
-// EncodingUnsupportedValueError
-//
-type EncodingUnsupportedValueError struct {
-	Value Value
-	Path  []string
-}
-
-func (e EncodingUnsupportedValueError) Error() string {
-	return fmt.Sprintf(
-		"encoding unsupported value to path [%s]: %[2]T, %[2]v",
-		strings.Join(e.Path, ","),
-		e.Value,
-	)
-}
```

Comment:
This is just a cleanup: Code producing this error was removed earlier in c3ac75224f3d585c95980bed71cb29e8fb3a0bea